src/main/kotlin/io/bitvavo/simulation/models/Order.kt

```diff
@@ -0,0 +1,33 @@
+package io.bitvavo.simulation.models
+
+class Order(
```

Comment:
You can use `data class`, it provides these methods out of the box

Reply:
I thought data class is meant to be an alternative of a record - which is for immutable data.
and this one is mutable. But indeed it does not seem to be the case, I can drop manual impl thanks.